code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -32,7 +32,7 @@
 		var/light_amount = 0 //How much light there is in the place, affects receiving nutrition and healing
 		if(isturf(loc)) //Else, there's considered to be no light
 			var/turf/T = loc
-			light_amount = T.get_lumcount(0.5) * 10
```

Comment:
I really don't know what the fuck this parameter does, it's some kind of shitty lerp that fucks the whole calculation up. I think it's from the way way old luminosity code or something. Either way these two are the last instances of get_lumcount() in the codebase with this parameter and they're getting fucked up because of it. I ran into the same problem with botany trays, which almost always believed themselves to be in the darkness due to this shit.